lxd/images.go

```diff
@@ -885,7 +885,19 @@ func autoUpdateImages(d *Daemon) {
 			continue
 		}
 
-		autoUpdateImage(d, nil, id, info)
+		// FIXME: since our APIs around image downloading don't support
+		//        cancelling, we run the function in a different
+		//        goroutine and simply abort when the context expires.
```

Comment:
The actual image download code does support cancellation as that's what we use to allow a user to cancel image downloads during container creation or image copy. But we probably need more plumbing to make this usable from here.

Reply:
Yeah.